packages/tables/backend/sirius-components-collaborative-tables/src/test/java/org/eclipse/sirius/components/collaborative/tables/handlers/EditCellEventHandlerTests.java

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************************
+ * Copyright (c) 2024 CEA LIST.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.tables.handlers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.collaborative.api.ChangeDescription;
+import org.eclipse.sirius.components.collaborative.tables.api.ITableQueryService;
+import org.eclipse.sirius.components.collaborative.tables.dto.EditCheckboxCellInput;
+import org.eclipse.sirius.components.collaborative.tables.messages.ICollaborativeTableMessageService;
+import org.eclipse.sirius.components.core.api.ErrorPayload;
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.core.api.IFeedbackMessageService;
+import org.eclipse.sirius.components.core.api.IPayload;
+import org.eclipse.sirius.components.tables.CheckboxCell;
+import org.eclipse.sirius.components.tables.Column;
+import org.eclipse.sirius.components.tables.ICell;
+import org.eclipse.sirius.components.tables.Line;
+import org.eclipse.sirius.components.tables.Table;
+import org.junit.jupiter.api.Test;
+
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import reactor.core.publisher.Sinks;
+
+/**
+ * Test for EditCellEventHandler.
+ *
+ * @author frouene
+ */
+public class EditCellEventHandlerTests {
```

Comment:
Create a proper integration tests instead of using mocks and testing only a subset of the edition

Reply:
This was to test an error case (no handler for the input).
Do we also want to test this kind of case via TI?